filament/src/materials/MaterialHelpersShading.fxh

```diff
@@ -126,11 +132,12 @@ void ApplyNormalMap(inout MaterialInputs material, inout FragmentData fragmentDa
         vec3 normalWS = TriplanarNormalMap(materialParams_normalTexture,
                                            materialParams.textureScaler.y,
                                            fragmentData.pos,
-                                           fragmentData.normal);
+                                           fragmentData.normal,
+                                           materialParams.normalIntensity);
         material.normal = normalWS * getWorldTangentFrame();
     }
     // By this time the normal should be ready, it is safe to apply the normal scale
-    material.normal.xy *= materialParams.normalIntensity;
+    //material.normal.xy *= materialParams.normalIntensity;
```

Comment:
Nit: you can delete this line, no need to keep it commented out.

Reply:
Fixed in [b7f849c](https://github.com/shapr3d/filament/pull/32/commits/b7f849c532856e32bfa86932ac41d942d7209fd8)